examples/model_selection/plot_learning_curve.py

```diff
@@ -181,7 +181,7 @@ def plot_learning_curve(
 title = "Learning Curves (Naive Bayes)"
 # Cross validation with 100 iterations to get smoother mean test and train
 # score curves, each time with 20% data randomly selected as a validation set.
-cv = ShuffleSplit(n_splits=100, test_size=0.2, random_state=0)
+cv = ShuffleSplit(n_splits=50, test_size=0.2, random_state=0)
```

Comment:
We need to update the comment above as well.

Reply:
Edited it. Thanks for the info @TomDLT